docs/source/notebooks/wps_climdex_days_demo.ipynb

```diff
@@ -0,0 +1,196 @@
+{
```

Comment:
<p>Should we turn this into a util method? We may need a more generalized version for other processes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pacificclimate/quail/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Oh ya good point, it would work for anthing that outputs a vector</p>